pkgs/development/compilers/z88dk/Makefile.patch

```diff
@@ -0,0 +1,60 @@
+--- /nix/store/lqr8s3yi8gn98gf3xybrmxg2dj1d4li2-source/Makefile	1970-01-01 01:00:01.000000000 +0100
```

Comment:
This patch looks odd to me, is it possible to avoid referring to store paths here? Perhaps it'd be better to modify the PREFIX define using sed or something instead.

Reply:
I don't see the problem here, this line is just to say it's a reference ... you can replace with what you want, the thing is that it a huge directory, so i don' t over copy but diff . I think it could be a good thing to have such store , it shows that you diff from the real nix fetched sources ... anyway ... the better is the worse ennemie of the good. Where should i past my time, fix random reviewer suggestion ...

Reply:
The issue is that you're checking in a fairly large diff compared to a single line of sed. What is more it seems the diff would have to be updated on each bump, given that it refers to specific store paths.

Reply:
i spend 2 days on that. I tried to fix the latest stable first until try to have working library in last time. It took very very long time to compile this stuff, each iteration is very long. It results in a 180MB of compiler + libraries. just to say, the value. I'm using it with caprice32 ... How i see the stuff : "Release early, release often" .

Reply:
Well, the patch will likely break the next time the version is bumped, unless you can show that patch will apply cleanly upon a version bump.

Reply:
done ...